completion/available/artisan.completion.bash

```diff
@@ -0,0 +1,27 @@
+# shellcheck shell=bash
+cite "about-completion"
+about-completion "Laravel artisan completion"
+
+# Completion function for Laravel artisan
+_artisan_completion() {
+	local cur commands
+	COMPREPLY=()
+	cur="${COMP_WORDS[COMP_CWORD]}"
+
+	# Only provide completions if artisan file exists in current directory or parent directories
+	if [[ ! -f "artisan" ]]; then
+		return 0
+	fi
+
+	# Get list of available artisan commands
+	# Use command prefix to bypass user aliases
+	commands=$(command php artisan --raw --no-ansi list 2> /dev/null | command sed "s/[[:space:]].*//g")
+
+	# shellcheck disable=SC2207
+	COMPREPLY=($(compgen -W "${commands}" -- "${cur}"))
```

Comment:
```suggestion
	COMPREPLY=($(compgen -W '${commands}' -- "${cur}"))
```

The option argument of `compgen -W` is subject to shell expansions like the `eval` arguments. It should be quoted by single-qutations to avoid extra expansions.

Reply:
interesting...